hyperspy/tests/drawing/test_mpl_testing_setup.py

```diff
@@ -16,14 +16,35 @@
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
 import matplotlib
-import pytest
 from distutils.version import LooseVersion
+import pytest
+
+from hyperspy.misc.test_utils import check_running_tests_in_CI
 
 
-def test_mlp_agg_for_testing():
-    assert matplotlib.get_backend() == 'agg'
+def test_mlp_agg_for_CI_testing():
```

Comment:
Why is this test needed? If everything works with something other than the agg backend there is nothing wrong about it...